pip/pep425tags.py

```diff
@@ -164,23 +164,39 @@ def get_supported(versions=None, noarch=False):
             # support macosx-10.6-intel on macosx-10.9-x86_64
             match = _osx_arch_pat.match(arch)
             if match:
```

Comment:
It would be nice to extract it in a `get_darwin_arches(platform)` function that could be tested...